### PR TITLE
Fixing issues introduced with new commands adding index

### PR DESCRIPTION
**Pull request by moxiegirl**
Adjust bullets
Fixing issues in command ordering. 
Adding index.md

Signed-off-by: Mary Anthony mary@docker.com

### CODE REVIEW

**Comment by moxiegirl** [comment]
ping @thaJeztah

**Comment by moxiegirl** [comment]
ping @docker/docs-owners

File: docs/reference/commandline/index.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+* [inspect](inspect)
+* [version](version)
+
+### Image commands
+
+* [build](build)
+* [commit](commit)
+* [history](history)
+* [images](images)
+* [import](import)
```
Comment (thaJeztah) [inline docs/reference/commandline/index.md]: import/export are counterparts so should both be here

File: docs/reference/commandline/index.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+* [search](search)
+
+### Network and connectivity commands
+
+* [network_connect](network_connect)
+* [network_create](network_create)
+* [network_disconnect](network_disconnect)
+* [network_inspect](network_inspect)
+* [network_ls](network_ls)
+* [network_rm](network_rm)
```
Comment (thaJeztah) [inline docs/reference/commandline/index.md]: These should probably be in a separate PR (don't think these are already in the docs yet?)

File: docs/reference/commandline/index.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+* [save](save)
+* [tag](tag)
+
+### Container commands
+
+* [attach](attach)
+* [cp](cp)
+* [create](create)
+* [diff](diff)
+* [events](events)
```
Comment (thaJeztah) [inline docs/reference/commandline/index.md]: events is probably "management"? not sure.

Was looking at the API docs what sections we used there: https://docs.docker.com/reference/api/docker_remote_api_v1.21/

Comment (moxiegirl) [inline docs/reference/commandline/index.md]: Events only reports container events. So, I think it is the right place. Management is more global...or more general about using Docker client/daemon.

File: docs/reference/commandline/index.md (hunk 2 above)
Comment (moxiegirl) [inline docs/reference/commandline/index.md]: That per merged last night....

**Comment by moxiegirl** [comment]
Author LGTM.  @thaJeztah if you find the comments entered ok, please feel free to merge. I'd like to get this in today I think we are coming up on an RC.

File: docs/reference/commandline/index.md (hunk 3 above)
Comment (thaJeztah) [inline docs/reference/commandline/index.md]: There was a PR for "build" events; I saw it was just turned down, so sgtm

**Comment by thaJeztah** [comment]
Yup, LGTM, thanks!

**Comment by moxiegirl** [comment]
Thanks Seb!
